src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,44 @@
-export const TodoInfo = () => {};
+import { Todo } from '../TodoList';
+import { User } from '../UserInfo';
+
+export const TodoInfo = ({
+  todos = [],
+  users,
+}: {
+  todos: Todo[];
+  users: User[];
+}) => {
+  const findUser = (userId: number) => {
+    const us = users.find(user => user.id === userId);
```

Comment:
Avoid using one-letter variable names like `us`. Consider renaming it to something more descriptive, such as `foundUser`, to improve code readability.